### PR TITLE
[kots]: add documentation on how to customize annotation, labels, envvars

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

Document the customization process

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/10941

## How to test

Follow documentation

## Release Notes

```release-note
[kots]: add documentation on how to customize annotation, labels, envvars
```

## Documentation

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2316"></a>

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2316"></a>

### CODE REVIEW

**Review by lucasvaltl — CHANGES_REQUESTED** [review]
Overall LGTM, just a couple smaller things to note.

File: gitpod/docs/self-hosted/latest/advanced/customization.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              value: world
+```
+
+In the event of multiple matches, the final matching customization is applied. Therefore, it is a good idea to structure your customization from least to most specific.
+
+> **Important:** System-generated values will never be overridden.
+
+## Limitations
+
+> **NOTE:** Your Gitpod installation may be inaccessible and unstable whilst this process takes place.
```
[the remark this answers is not in the document]
Comment (MrSimonEmms) [reply gitpod/docs/self-hosted/latest/advanced/customization.md:113]: Yeah, it's a bit unclear. I'm going to rewrite this section

File: gitpod/docs/self-hosted/latest/advanced/customization.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+---
+section: self-hosted/latest
```
Comment (Siddhant-K-code) [inline gitpod/docs/self-hosted/latest/advanced/customization.md:2]: will there be any `subsection` for this?

Comment (MrSimonEmms) [reply gitpod/docs/self-hosted/latest/advanced/customization.md:2]: I don't think so. I based it upon the [Air Gap](https://github.com/gitpod-io/website/blob/main/gitpod/docs/self-hosted/latest/advanced/air-gap.md) page in the advanced section

**Review by lucasvaltl — APPROVED** [review]
LGTM!

**Pull request merged**